traits/constants.py

```diff
@@ -110,12 +110,15 @@ class ComparisonMode(IntEnum):
     Indicates when trait change notifications should be generated based upon
     the result of comparing the old and new values of a trait assignment:
 
-    * 0 (no_compare): The values are not compared and a trait change
-        notification is generated on each assignment.
-    * 1 (object_id_compare): A trait change notification is
-        generated if the old and new values are not the same object.
-    * 2 (equality_compare): A trait change notification is generated if the
-        old and new values are not equal using Python's standard equality
+    0 (no_compare)
```

Comment:
@corranwebster What do you think about removing the numbers here altogether? I think they can reasonably be regarded as an implementation detail, and there's always the source link in the documentation for people who want to see the actual numbers.

Reply:
Yes, I'd be quite happy with that.